scripts/typescript-compatibility-build.yml

```diff
@@ -0,0 +1,120 @@
+trigger:
+  branches:
+    include:
+      - tsc-compat-debug
+      #- refs/tags/office-ui-fabric-react_v*
+
+variables:
+  - name: PackageName
+    value: office-ui-fabric-react
+  - name: CompatRepoUrl
+    value: https://github.com/Microsoft/ui-fabric-ts-validation.git
+  - name: NodeVersion
+    value: '10.x'
+  - name: VmImage
+    value: 'Ubuntu 16.04'
+
+jobs:
+  - job: Build
+    pool:
+      vmImage: $(VmImage)
+    timeoutInMinutes: 60
+    steps:
+      - checkout: self
+        clean: true
+
+      - task: NodeTool@0
+        displayName: 'Use Node $(NodeVersion)'
+        inputs:
+          versionSpec: '$(NodeVersion)'
+          checkLatest: true
+
+      # Install Rush repo dependencies
+      - script: |
+          node common/scripts/install-run-rush.js install --bypass-policy
```

Comment:
Love the granular steps with descriptive names!

Reply:
Thank you! 🙇‍♂️